queries.sql

```diff
@@ -7,4 +7,11 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered,SUM(escape_attempts) FROM animals GROUP BY neutered ORDER BY sum DESC;
```

Comment:
- [ ] Kindly make sure the query to answer this question: **"Who escapes the most, neutered or not neutered animals?"(`line 15`)** gives the correct answer(i.e it should give the animal(s) with the most escape_attempts, neutered or not neutered).